toolchain/check/context.h

```diff
@@ -119,6 +120,10 @@ class Context {
 
   auto scope_stack() -> ScopeStack& { return scope_stack_; }
 
+  auto deferred_definition_scope_stack() -> DeferredDefinitionScopeStack& {
+    return deferred_definition_scope_stack_;
+  }
```

Comment:
Maybe move this beneath the `scope_stack` member accessors? (i.e., break_continue_stack and full_pattern_stack)